src/app/create/basicauction/page.tsx

```diff
@@ -0,0 +1,225 @@
+/* eslint-disable react/jsx-props-no-spreading */
+
+'use client';
+
+import { SubmitHandler, useFieldArray, useForm } from 'react-hook-form';
+import DeleteIcon from '@/assets/svg/delete.svg';
+import ErrorIcon from '@/assets/svg/error.svg';
+import { useRef } from 'react';
+
+import * as S from '@/app/create/basicauction/Basicauction.css';
+
+type ImageUpload = {
+  file: File;
+};
+
+type AuctionInputs = {
+  nameRequired: string;
+  startPriceRequired: number;
+  imageRequired: ImageUpload[];
+  infoRequired: string;
+  dateRequired: string;
+  timeRequired: string;
+};
+
+const MAX_CONTENT = 500;
+
+export default function Home() {
+  const {
+    register,
+    control,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<AuctionInputs>();
+
+  const { fields, append, remove } = useFieldArray({
+    control,
+    name: 'imageRequired',
+    keyName: 'imageRequiredId',
+    rules: {
+      required: '이미지를 업로드해 주세요.',
+    },
+  });
+
+  const inputFile = useRef(null);
+  const infoRequired = watch('infoRequired');
+  const endDate = watch('dateRequired');
+  const endTime = watch('timeRequired');
+
+  const addImage = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files) {
+      const uploadFile = Array.from(e.target.files);
+      const files = uploadFile.map(file => ({ file }));
+      append(files);
+    }
+  };
+
+  const onSubmit: SubmitHandler<AuctionInputs> = data => console.log(data);
+
+  return (
+    <form className={S.auctionForm} onSubmit={handleSubmit(onSubmit)}>
+      <label htmlFor="name" className={S.auctionLabel}>
+        <div className={S.title}>상품명</div>
+        <input
+          className={S.auctionInput}
+          id="name"
+          type="text"
+          placeholder="상품명"
+          {...register('nameRequired', { required: '상품명을 입력해 주세요.' })}
+        />
+        {errors.nameRequired && (
+          <span className={S.error}>
+            <ErrorIcon />
+            {errors.nameRequired.message}
+          </span>
+        )}
+      </label>
+      <label htmlFor="startPrice" className={S.auctionLabel}>
+        <div className={S.title}>시작가</div>
+        <input
+          className={`${S.auctionInput} ${S.priceInput}`}
+          id="startPrice"
+          type="number"
+          placeholder="원"
```

Comment:
```suggestion
          placeholder="₩"
```
으로 변경하고 상품명과 동일하게 왼쪽에서 시작하는 것은어떤가요?